synapse/handlers/pagination.py

```diff
@@ -70,6 +121,9 @@ class PaginationHandler:
     paginating during a purge.
     """
 
+    # remove the purge from the list 24 hours after it completes
+    CLEAR_PURGE_TIME = 3600 * 24
```

Comment:
we tend to use milliseconds rather than seconds.

```suggestion
    # when to remove a completed deletion/purge from the results map
    CLEAR_PURGE_AFTER_MS = 1000 * 3600 * 24   # 24 hours
```